image TestCreateRegistrySaveError is flaky

```
ok      github.com/openshift/origin/pkg/image/registry/etcd 1.094s  coverage: 96.0% of statements
--- FAIL: TestCreateRegistrySaveError (0.00 seconds)
    rest_test.go:134: Expected failure status, got &{{%!V(string=) %!V(string=) {{%!V(int64=0) %!V(uintptr=0) %!V(*time.Location=<nil>)}} %!V(string=) %!V(uint64=0) %!V(string=) %!V(string=)} %!V(string=Failure) %!V(string=test error) %!V(api.StatusReason=) %!V(*api.StatusDetails=<nil>) %!V(int=500)}
FAIL
FAIL    github.com/openshift/origin/pkg/image/registry/image    0.099s
ok      github.com/openshift/origin/pkg/image/registry/imagerepository  1.137s  coverage: 86.5% of statements
```

--- a/pkg/image/registry/image/rest_test.go
+++ b/pkg/image/registry/image/rest_test.go
@@ -130,12 +130,11 @@ func TestCreateRegistrySaveError(t *testing.T) {
 		if !ok {
 			t.Errorf("Expected status type, got: %#v", result)
 		}
-		if status.Status != kubeapi.StatusFailure || status.Message != "foo" {
-			t.Errorf("Expected failure status, got %#V", status)
+		if status.Status != kubeapi.StatusFailure || status.Message != "test error" {
+			t.Errorf("Expected failure status, got %#v", status)
 		}
 	case <-time.After(50 * time.Millisecond):
 		t.Errorf("Timed out waiting for result")
-	default:
 	}
 }
 
@@ -165,7 +164,6 @@ func TestCreateImageOK(t *testing.T) {
 		}
 	case <-time.After(50 * time.Millisecond):
 		t.Errorf("Timed out waiting for result")
-	default:
 	}
 }
 
@@ -239,6 +237,5 @@ func TestDeleteImage(t *testing.T) {
 		}
 	case <-time.After(50 * time.Millisecond):
 		t.Errorf("Timed out waiting for result")
-	default:
 	}
 }

